src/main/java/org/infernalstudios/shieldexp/init/Config.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright 2022 Infernal Studios
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.infernalstudios.shieldexp.init;
+
+import net.minecraft.item.Item;
+import net.minecraftforge.common.ForgeConfigSpec;
+import net.minecraftforge.registries.ForgeRegistries;
+
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class Config {
```

Comment:
This should use our [Blazing Config](https://github.com/infernalstudios/Blazing-Config), but don't change it. This will be done later on.